packages/ui/src/utils/index.ts

```diff
@@ -26,16 +26,16 @@ export const getSearchParameters = (parameters: string | URLSearchParams, key: s
   return searchParameters.get(key) ?? undefined;
 };
 
-type Entries<T> = Array<
+export type Entries<T> = Array<
   {
     [K in keyof T]: [K, T[K]];
   }[keyof T]
 >;
 
+// eslint-disable-next-line no-restricted-syntax
```

Comment:
or we can have a `fromEntries()` as well if the type issue cannot fix

Reply:
updated